collect_app/src/main/java/org/odk/collect/android/database/DatabaseFormsRepository.java

```diff
@@ -48,6 +48,15 @@ public DatabaseFormsRepository() {
         }
     }
 
+    @Override
+    public List<Form> getAll(String jrFormId, @Nullable String jrVersion) {
```

Comment:
I think we should add tests for these new repo methods so we then know the fake and real implementation have consistent behaviours.

Reply:
Thanks for the nudge, that's an improvement.